### PR TITLE
Update Chinese docs

### PR DESCRIPTION
**Pull request by vickyhella**
- Update Chinese translation for v1.2 and v1.1

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/airgap.md [hunk 1]
```diff
@@ -36,8 +36,20 @@ Harvester ISO 镜像包括实现在离线环境中运行的所有包。
...
 
 Harvester 在用户配置的 `no-proxy` 后附加必要的地址，来确保内部流量能正常工作。
-例如：`localhost,127.0.0.1,0.0.0.0,10.0.0.0/8,longhorn-system,cattle-system,cattle-system.svc,.svc,.cluster.local`
+例如：`localhost,127.0.0.1,0.0.0.0,10.0.0.0/8,longhorn-system,cattle-system,cattle-system.svc,harvester-system,.svc,.cluster.local`。v1.1.2 开始将 `harvester-system` 添加到列表。
 
 如果集群中的节点之间不使用代理进行通信，你需要在第一个节点安装成功后将 CIDR 添加到 `http-proxy.noProxy`。请参阅[部署多节点集群失败](./troubleshooting/harvester.md#http-proxy-设置错误导致多节点集群部署失败)。
 
 :::
+
+## 来宾集群镜像
```
[the remark this answers is not in the document]
Comment (guangbochen) [reply i18n/zh/docusaurus-plugin-content-docs/current/airgap.md:45]: and ditto for the below.

File: i18n/zh/docusaurus-plugin-content-docs/current/host/host.md [hunk 2]
```diff
@@ -119,6 +119,55 @@ Admin 用户可以点击 **Enable Maintenance Mode** 来自动驱逐节点中所
...
+![Check the removing disk](/img/v1.2/host/remove-disks-longhorn-nodes.png)
+
+4. 选择 **Edit node and disks**。
+
+![Edit node and disks](/img/v1.2/host/remove-disks-longhorn-nodes-edit.png)
+
+5. 滚动到要移除的磁盘。
+- 将 `Scheduling` 设置为 `Disable`。
+- 将 `Eviction Requested` 设置为 `True`。
+   选择 **Save**。不要选择删除图标。
```
Comment (guangbochen) [inline i18n/zh/docusaurus-plugin-content-docs/current/host/host.md:153]: this should be a new line here.

File: i18n/zh/docusaurus-plugin-content-docs/current/airgap.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vickyhella) [reply i18n/zh/docusaurus-plugin-content-docs/current/airgap.md:45]: Changed all occurrences of `来宾` to `Guest`

**Review by guangbochen — APPROVED** [review]
LGTM, thanks

**Pull request merged**